test/fixtures/controller.polarArea/angle-array.json

```diff
@@ -0,0 +1,32 @@
+{
+    "debug": true,
```

Comment:
Can be removed, it's only used for debugging and is likely the reason why unit tests are failing on Travis.